psst-gui/src/controller/playback.rs

```diff
@@ -39,22 +42,18 @@ pub struct PlaybackController {
     media_controls: Option<MediaControls>,
     has_scrobbled: bool,
     scrobbler: Option<Scrobbler>,
+    discord_rpc_sender: Option<Sender<DiscordRpcCmd>>,
     startup: bool,
 }
-fn init_scrobbler_instance(
-    data: &AppState
-) -> Option<Scrobbler> {
+fn init_scrobbler_instance(data: &AppState) -> Option<Scrobbler> {
     if data.config.lastfm_enable {
         if let (Some(api_key), Some(api_secret), Some(session_key)) = (
             data.config.lastfm_api_key.as_deref(),
             data.config.lastfm_api_secret.as_deref(),
             data.config.lastfm_session_key.as_deref(),
         ) {
-            match LastFmClient::create_scrobbler(
-                Some(api_key),
-                Some(api_secret),
-                Some(session_key),
-            ) {
+            match LastFmClient::create_scrobbler(Some(api_key), Some(api_secret), Some(session_key))
+            {
```

Comment:
There are a few possibly unintended linter corrected diffs like this?

Reply:
I’m not sure what linter the person who originally implemented that used. In my case, rustfmt was automatically run in Zed with the project's current configuration file. I didn’t manually adjust the formatting. Any changes would have come from that.